addons/survey/views/survey_templates_statistics.xml

```diff
@@ -73,10 +93,15 @@
                             </t>
                             <!-- Scoring info -->
                             <t t-if="question_scoring">
-                                <span class="badge badge-success ml-3"><span t-esc="len(question_data['right_inputs'])"></span> Correct</span>
-                                <span class="badge badge-warning ml-1" t-if="question.question_type == 'multiple_choice'">
-                                    <span t-esc="len(question_data['partial_inputs'])"></span> Partial
-                                </span>
+                                <t t-if="question.question_type in ['simple_choice', 'multiple_choice']">
+                                    <span class="badge badge-success ml-3"><span t-esc="len(question_data['right_inputs'])"></span> Correct</span>
+                                    <span class="badge badge-warning ml-1" t-if="question.question_type == 'multiple_choice'">
+                                        <span t-esc="len(question_data['partial_inputs'])"></span> Partial
+                                    </span>
+                                </t>
+                                <t t-if="question.is_scored_question">
+                                    <span class="badge badge-success ml-3"><span t-esc="len(question_data['right_inputs'])"></span> Correct</span>
```

Comment:
I think we could easily add an "incorrect" badge here.

Also, it looks like we only need the number of right_inputs.
Maybe rename to `correct_inputs_count` and only send the len() to the template?

Reply:
Updated to `correct_inputs_count` as well as `partial_inputs_count` to avoid having len() in the template. The same could probably be applied to some of the `survey_question._prepare_statistics()` badge values of "Answered" and "Skipped" depending on how the fix for the counting of skipped choice questions is done + for how an "Incorrect" badge is added in (not done as part of this task since requires more changes than expected to be done cleanly).